fury/data/fetcher.py

```diff
@@ -156,22 +167,21 @@ def fetch_data(files, folder, data_size=None):
 
     all_skip = True
     for f in files:
-        url, md5 = files[f]
+        url, sha = files[f]
         fullpath = pjoin(folder, f)
-        if os.path.exists(fullpath) and (_get_file_md5(fullpath) == md5):
+        if os.path.exists(fullpath) and (_get_file_sha(fullpath) == sha):
```

Comment:
`sha.lower()` instead of  `sha` because it seems that `_get_file_sha` returns a lower-case sha and the stored one is upper case